src/app/components/BookList/BookList.jsx

```diff
@@ -4,16 +4,28 @@ import PropTypes from 'prop-types';
 import About from '../About/About';
 import Book from '../Book/Book';
 
-const BookList = ({ isJsEnabled, picks, listType }) => {
+const BookList = ({
+  isJsEnabled,
+  picks,
+  type,
+  displayInfo,
+  picksCount,
+}) => {
   const renderBookItems = currentBooks => (
     currentBooks.length ?
-      currentBooks.map((book, i) => <Book key={i} pick={book} isJsEnabled={isJsEnabled} />)
+      currentBooks.map(book => <Book key={book.slug} pick={book} isJsEnabled={isJsEnabled} />)
       : null
   );
 
+  const { displayDate = {}, displayAge } = displayInfo;
+  const booksfound = `${picksCount} Book${picksCount === 1 ? '' : 's'} Found`;
+
   return (
     <div className="booklist-section nypl-column-three-quarters">
-      <h2>2017 Picks</h2>
+      <h2 id="list-title" tabIndex="0">
```

Comment:
@ktp242 I'll extract this out into another component so you can use it for the sidebar.

Reply:
Thanks!